flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniCluster.java

```diff
@@ -757,7 +758,7 @@ protected RpcService createRpcService(
 
 		final ActorSystem actorSystem = AkkaUtils.createActorSystem(effectiveAkkaConfig);
 
-		return new AkkaRpcService(actorSystem, askTimeout);
+		return new AkkaRpcService(actorSystem, AkkaRpcServiceConfiguration.fromConfiguration(configuration));
```

Comment:
I would suggest to replace the `askTimeout` parameter with a `AkkaRpcServiceConfiguration akkaRpcServiceConfiguration` parameter and pass it to this method.